apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/pac/AbstractR4ResourceProvider.java

```diff
@@ -197,7 +207,7 @@ public T read(@IdParam IdType claimId, RequestDetails requestDetails) {
     }
     final boolean includeTaxNumbers = returnIncludeTaxNumbers(requestDetails);
 
-    if (claimId == null) throw new IllegalArgumentException("Resource ID can not be null");
+    //    if (claimId == null) throw new IllegalArgumentException("Resource ID can not be null");
```

Comment:
Do we need this change?

Reply:
Let's not comment out code. If we don't need it, remove it. We can always find it in the git history if needed.